src/main/java/com/github/devotedmc/hiddenore/Config.java

```diff
@@ -65,18 +70,24 @@ public static void loadConfig() {
 					//TODO pretty sure this whole scope here doesn't work.
 					//TODO double check if this is supposed to be the specific config or default-world.yml for the reader. I'm too tired to be able to follow this around in my head right now.
 					HiddenOre.getPlugin().getLogger().info("Loading configs for world " + world.getName());
-					Reader worldExistsReader = new InputStreamReader(HiddenOre.getPlugin().getResource(String.format("%s-config.yml", world.getName())));
-					configurations.put(world.getName(), YamlConfiguration.loadConfiguration(worldExistsReader));
+					try(Reader worldExistsReader = new FileReader(saveFile)){
+						configurations.put(world.getName(), YamlConfiguration.loadConfiguration(worldExistsReader));
+					}catch(IOException ioe){
+						HiddenOre.getPlugin().getLogger().warning(String.format("A YAML file could not be found for the world named %s.", world.getName()));
+					}
 					//make sure that you put a file there first so it doesnt null pointer or just fix the routing dude
 					//configurations.get(world.getName()).load(saveFile);
 					HiddenOre.getPlugin().getLogger().info("Loaded Configs for world " + world.getName());
 				}else{
 					HiddenOre.getPlugin().getLogger().info("Configs do not exist for world " + world.getName() + ", generating them...");
 					File defaultWorldFile = new File(HiddenOre.getPlugin().getDataFolder(), "default-world.yml");
```

Comment:
Not to beat a horse, but, I'm 60% sure the literal file default-world.yml won't exist on the actual disk-filesystem, but only in the "JAR" filesystem. To access the JAR filesystem, use the .getResource() syntax. Checking the data folder is unlikely to find the default file which will never be "unpacked" there...